p4-16/psa/PSA.mdk

```diff
@@ -1095,6 +1095,27 @@ Rationale:
   accessed from only a single pipeline on such a device.
 
 
+## PSA Table Properties
+
+Table [#table-table-properties] lists all P4 table properties defined
+by PSA that are not included in the base P4~16~ language
+specification.
+
+~ TableFigure {#table-table-properties; \
+  caption: Summary of PSA table properties.}
+|-----------------------|------|---------------------------------------|
+| Property name         | Type | See also |
++:----------------------+:-----+:--------------------------------------+
+| `psa_direct_counters` | list of `DirectCounter` instance names | Section [#sec-direct-counter] |
+|-----------------------|----------------------------------------------|
+| `psa_direct_meters` | list of `DirectMeter` instance names | Section [#sec-meters] |
+|-----------------------|----------------------------------------------|
+| `implementation`      | instance name of one `ActionProfile` | Sections [#sec-action-profile], [#sec-action-selector] |
```

Comment:
Should we consider a different name (e.g. psa_implementation)?

Reply:
Updated this PR to change this occurrence of `implementation`, and other occurrences in example code and the action profile/selector text that I could find, to `psa_implementation`.